windows_containers/wmco_rn/windows-containers-release-notes-10-15-x-known-limitations.adoc

```diff
@@ -0,0 +1,39 @@
+:_mod-docs-content-type: ASSEMBLY
```

Comment:
This looks like you are adding information in the new assembly directly. This is a tad nonstandard and also you don't seem to be removing the module where it was before? Might this confuse someone in the future who might need to edit the information - they might edit the module and be surprised the change does not go in?

Or is the module actually removed and the PR UI just does not show it?